src/wmnds/components/textarea/_textarea.njk

```diff
@@ -0,0 +1,64 @@
+{% macro wmndsTextarea(params) %}
```

Comment:
Could you put this folder back into the form-elements for now?

Reply:
Yeah, just did it a few minutes ago! 👍  We are synchronized!